storage/s3.go

```diff
@@ -347,6 +347,14 @@ func (s *S3) Copy(ctx context.Context, from, to *url.URL, metadata Metadata) err
 	return err
 }
 
+func escapeKey(sourceKey string) string {
```

Comment:
Could you move this function into our `url` package?

Reply:
Moved to `url.go`

https://github.com/peak/s5cmd/blob/cd5170dd8c7abad0d75de47335a7f1def3711fc1/storage/url/url.go#L352-L359

Reply:
👍